### PR TITLE
docs: add docs for standards object

### PR DESCRIPTION
**Pull request by straker**
## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Follows the commit message policy, appropriate for next version
- [x] Code is reviewed for security

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -176,6 +176,7 @@ axe.configure({
 	reporter: 'option',
 	checks: [Object],
 	rules: [Object],
+	standards: [Object],
```
Comment (stephenmathieson) [inline doc/API.md:179]: I know you're just following convention here, but this reads as "_an array of objects_", which I do not believe is correct.

Comment (straker) [reply doc/API.md]: Ops, you're right. Shouldn't have the []

File: doc/standards-object.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+- `values` - array(required for only `mntoken` and `mntokens`). The list of valid values for the attribute.
+- `allowEmpty` - boolean(optional, default `false`). If the attribute is allowed to have no value.
+- `global` - boolean(optional, default `false`). If the attribute is a [global ARIA attribute](https://www.w3.org/TR/wai-aria-1.1/#global_states).
+- `unsupported` - boolean(optional, default `false`). If the attribute is unsupported. Use this property to disable an attribute.
+
+## Aria Roles
+
+The [`ariaRoles`](../lib/standards/aria-roles.js) object defines valid ARIA roles and their use.
+
+## Used by Rules
```
Comment (WilcoFiers) [inline doc/standards-object.md:60]:
```suggestion
### Used by Rules
```

File: doc/standards-object.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  - `structure`
+  - `landmark`
+- `requiredContext` - array(optional). List of required parent roles.
+- `requiredOwned` - array(optional). List of required owned roles.
+- `requiredAttrs` - array(optional). List of required attributes.
+- `allowedAttrs` - array(optional). List of allowed attributes (besides any required and global ARIA attributes).
+- `nameFromContent` - boolean(optional. Default `false`). If the role allows name from content when calculating the accessible name.
+- `unsupported` - boolean(optional. Default `false`). If the role role is unsupported. Use this property to disable a role.
+
+## Dpub Roles
```
Comment (WilcoFiers) [inline doc/standards-object.md:82]:
```suggestion
### Dpub Roles
```

File: doc/standards-object.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+- `shadowRoot` - boolean(optional. Default `false`). If the element is allowed to have a shadow root.
+- `implicitAttrs` - object(optional. Default `{}`). Any implicit ARIA attributes for the element and their default value.
+- `namingMethods` - array(optional. Default `[]`). The [native text method](../lib/commons/text/native-text-methods.js) used to calculate the accessible name of the element.
+- `variant` - object(optional). Sometimes an element can have different properties based on which attributes are defined on it. For example, an `img` is only considered interactive if it has the `usemap` attribute.
+
+  For these cases, the `variant` property allows different definitions for the element. The `variant` object gives a name to each variant, defines when the variant applies through the `matches` property (a `matcher` object), and provides a `default` variant in the case none of the other variants match. A variant should only define what can change based on the attributes. Any properties that always apply regardless of attributes should be defined at the top level.
```
Comment (WilcoFiers) [inline doc/standards-object.md:112]: Should we have a separate section for Variant that explains this, instead of having it all be part of this listitem?

**Pull request merged**